tools/gulptasks/jsdoc-dts.js

```diff
@@ -46,10 +46,11 @@ function jsDocESMDTS() {
         .getFilePaths('code', true)
         .filter(file => (
             file.endsWith('.d.ts') &&
-            !file.split(path.sep).includes('dashboards') &&
-            !file.split(path.sep).includes('grid') &&
-            !file.split(path.sep).includes('es-modules') &&
-            !file.split(path.sep).includes('esm')
+            !file.includes('dashboards') &&
+            !file.includes('datagrid') &&
+            !file.includes('es-modules') &&
+            !file.includes('esm') &&
+            !file.includes('grid')
```

Comment:
The "split" version was an upgrade. Has something changed or should this be change?

Reply:
Sorry, I overlooked the detail when resolving merge conflicts.